src/checkers/jetbrainschecker.py

```diff
@@ -48,3 +54,11 @@ async def check(self, external_data: ExternalBase):
         )
 
         external_data.set_new_version(new_version)
+
+    @staticmethod
+    def _get_download_node_name(arch: str) -> str:
```

Comment:
I think this static mapping should be a class property.

Reply:
i don't think property would work as I need to access external data argument passed to check method. I think this would still be a function.

Reply:
Your initial submission (before commit 5c96449a30c659bec15f1b631d36a3668e7dcb47) was mostly correct, you should've just accessed the class property inside a method as `self.DOWNLOAD_NODE_SUFFIX`, not just `DOWNLOAD_NODE_SUFFIX`.

Reply:
Got it. I'll swap it out.

Reply:
I simplified the code and made download node suffix static.

Reply:
Please let me know if you would like me to make any other changes. Thanks!